collection-scripts/gather_network_logs

```diff
@@ -237,6 +242,22 @@ function gather_kuryr_nodes_data {
   done
 }
 
+function collect_node_log_files {
+# Call format: NODE [file_paths]
+# Collects log files from given paths,
+# path should be relative to the node's /var/logs/ folder,
+# e.g. openvswitch/ovs-vswitchd.log will be collected to "$NETWORK_LOG_PATH"/"$NODE"_ovs-vswitchd.log
```

Comment:
we already capture ovs-vswitchd.log as part of must-gather now:
https://github.com/openshift/must-gather/blob/cbf587fe6e4c97196e84aea68ad7a57210035e92/collection-scripts/gather_service_logs#L9

The issue is we need to set the logging levels in it, by doing ovs-vsctl vlog/set INFO for example. It looks like we just use to set it to INFO as shown here:
https://github.com/openshift/cluster-network-operator/blob/release-4.6/bindata/network/ovn-kubernetes/006-ovs-node.yaml#L92

We previously ran this OVS container in 4.6 that took care of this, but we no longer have it since 4.7. So log levels are not getting set correctly. I think you will need a PR in CNO to set the log level for OVS in the ovnkube-node template:
https://github.com/openshift/cluster-network-operator/blob/release-4.6/bindata/network/ovn-kubernetes/ovnkube-node.yaml

or alternatively you could put it in MCO:
https://github.com/openshift/machine-config-operator/blob/master/templates/common/_base/files/configure-ovs-network.yaml

Reply:
Thanks for the comment, I will open different PR for the bug fix, and leave minor fixes in this one